sycl/include/sycl/ext/oneapi/experimental/current_device.hpp

```diff
@@ -15,8 +15,15 @@ inline namespace _V1 {
 namespace ext::oneapi::experimental::this_thread {
 
 namespace detail {
-inline sycl::device &get_current_device_ref() {
-  static thread_local sycl::device current_device{sycl::default_selector_v};
```

Comment:
Note that could extend `sycl::device`'s lifetime past `global_handler` destruction when running our unittests, but that device wasn't really usable because we've already released all the UR resources at that point.